pytest-selenium/pages/content.py

```diff
@@ -78,6 +116,68 @@ def click_toc_toggle_button(self):
                     expected.invisibility_of_element_located(self.toc_toggle_button)
                 )
 
+        class TableOfContents(Region):
+            _root_locator = (By.CSS_SELECTOR, '[data-testid="toc"]')
+            _chapter_toggle = (By.CSS_SELECTOR, "ol li details")
+            _toc_chapter_name_locator = (By.XPATH, "(//ol/li/details/summary/div/span)")
+
+            @property
+            def is_chapter_expanded(self):
+                return self.find_element(*self._chapter_toggle).get_attribute("open")
+
+            @property
+            def number_of_chapters(self):
+                return len(self.find_elements(*self._chapter_toggle))
+
+            @property
+            def chapters(self):
```

Comment:
you have this organized such that the toc has chapters and the chapter have pages but that is not true, the toc can have pages directly and the chapters can have more chapters inside

Reply:
TOC is removed from this PR. But I will keep this point in mind when doing TOC